packages/date-picker/docs/Calendar.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react"
+import { Story } from "@storybook/react"
+import { usePopper } from "react-popper"
+import { enAU } from "date-fns/locale"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import { DayOfWeek } from "../src/DatePicker"
+import { Calendar, CalendarProps } from "../src/DatePicker/components/Calendar"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+
+const SUPPORTED_LOCALES = ["en-US", "en-AU"]
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.datePicker}/Primitives/Calendar`,
+  component: Calendar,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          "This is a subcomponent - use DatePicker or DateRangePicker.",
+      },
+    },
+  },
+  argTypes: {
+    locale: {
+      options: SUPPORTED_LOCALES,
+      control: { type: "radio" },
+    },
+  },
+}
+
+const CalendarExample = (props: Partial<CalendarProps>): JSX.Element => {
+  const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(
+    null
+  )
+
+  const [referenceElement, setReferenceElement] =
+    useState<HTMLDivElement | null>(null)
+
+  const { styles, attributes } = usePopper(referenceElement, popperElement, {
+    modifiers: [
+      {
+        name: "offset",
+        options: {
+          offset: [0, 15],
+        },
+      },
+    ],
+    placement: "bottom-start",
+  })
+
+  return (
+    <div ref={setReferenceElement}>
+      <Calendar
+        mode="single"
+        id={props.id ? props.id : "calendar-dialog"}
```

Comment:
```suggestion
        id={props.id}
```
I take it this doesn't work? 😅

Reply:
Yeah it doesn't I tried - ts error